src/OpenTelemetry.ResourceDetectors.Host/README.md

```diff
@@ -0,0 +1,39 @@
+# Host Resource Detectors
+
+[![NuGet version badge](https://img.shields.io/nuget/v/OpenTelemetry.ResourceDetectors.Host)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.Host)
+[![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.ResourceDetectors.Host)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.Host)
+
+## Getting Started
+
+You need to install the
+`OpenTelemetry.ResourceDetectors.Host` package to be able to use the
+Host Resource Detectors.
+
+```shell
+dotnet add package OpenTelemetry.ResourceDetectors.Host --prerelease
+```
+
+## Usage
+
+You can configure Host resource detector to
+the `TracerProvider` with the following example below.
```

Comment:
same comment as https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1506#discussion_r1432922740

Reply:
I will create follow issue for this. I think we should revisit all detectors at once and make the documentation consistent.

Reply:
#1509